modules/logging/aws-for-fluent-bit/main.tf

```diff
@@ -0,0 +1,116 @@
+resource "kubernetes_namespace" "this" {
+  count = var.create_namespace ? 1 : 0
+  depends_on = [
+    var.module_depends_on
+  ]
+
+  metadata {
+    name = var.namespace_name
+  }
+}
+
+data "aws_iam_policy_document" "aws_for_fluent_bit_assume_role_policy" {
+  statement {
+    actions = ["sts:AssumeRoleWithWebIdentity"]
+    effect  = "Allow"
+
+    condition {
+      test     = "StringEquals"
+      variable = "${replace(var.cluster_oidc_url, "https://", "")}:sub"
+      values   = ["system:serviceaccount:${local.namespace}:${var.service_account_name}"]
+    }
+
+    principals {
+      identifiers = [var.cluster_oidc_arn]
+      type        = "Federated"
+    }
+  }
+}
+
+resource "aws_iam_role" "aws_for_fluent_bit_iam_role" {
+  name               = "${var.cluster_name}-aws-for-fluent-bit"
+  assume_role_policy = data.aws_iam_policy_document.aws_for_fluent_bit_assume_role_policy.json
+
+  tags = {
+    Environment = var.environment
+    Project     = var.project
+  }
+}
```

Comment:
Could you please move this sections to iam.tf?

Reply:
Done